packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -290,20 +290,9 @@ const getEnvName = async (context: $TSContext): Promise<string> => {
   return envName;
 };
 
-const isNewEnv = (envName: string): boolean => {
-  let newEnv = true;
-  const projectPath = process.cwd();
-  const teamProviderInfo = stateManager.getTeamProviderInfo(projectPath, {
-    throwIfNotExist: false,
-    default: {},
-  });
-
-  if (teamProviderInfo[envName]) {
-    newEnv = false;
-  }
-
-  return newEnv;
-};
+const isNewEnv = (envName: string): boolean => !Object.keys(
+  stateManager.getLocalAWSInfo(process.cwd(), { throwIfNotExist: false, default: {} }),
```

Comment:
```suggestion
  stateManager.getLocalEnvInfo().envName
```

Reply:
I don't think that check is exactly the same, because this could be pulling an environment that already exists but isn't necessarily the current environment

Reply:
Thats strange!!!